pages/02-01-up-and-running.md

```diff
@@ -22,15 +22,93 @@ Along with installing LÖVE, you will need a text editor for creating Lua files
 
 When you launch LÖVE, (see instructions below on how to do that) you will be greeted with a friendly graphic and the text "NO GAME", meaning you are running the engine but didn't give it a game to load.
 
-### macOS and other posix systems
+## macOS
+
+Once you have downloaded the LÖVE binary for macOS (64-bit zipped), proceed to the "Downloads" folder and unzip the archive. You should now see an application called "love".
+
+![macOS-love](/images/02-01-macos-folder.png)
+
+macOS might show you a warning modal, because you are trying to open an application by an unverified developer. If so, right click on the application and choose "open" and "open" again in the following dialog. You should now be greeted by the no-game screen.
+
+### Starting your game
```

Comment:
The way gitbook renders h3 from h2 it is hard to tell them apart, so since this is a subheader to `### macOS` try using `**Bold text**` instead.

Reply:
Fixed in latest commit.